Resources/config/oauth.xml

```diff
@@ -37,7 +37,7 @@
         </service>
 
         <service id="fos_oauth_server.create_client_command" class="FOS\OAuthServerBundle\Command\CreateClientCommand">
-            <argument type="service" id="fos_oauth_server.client_manager.default" />
+            <argument type="service" id="fos_oauth_server.client_manager" />
```

Comment:
Why was this required?

Reply:
It is driver implementation detail. The bundle should use services without `.default` postfix. It follows the rule in other places for other services. This is the only place where the rule is broken.